test/adapter/json_api/links_test.rb

```diff
@@ -73,7 +79,10 @@ def test_resource_links
                 stuff: 'value'
               }
             },
+            author: 'http://example.com/link_authors/1337',
+            link_authors: 'http://example.com/link_authors',
             other: '//example.com/resource',
+            posts: 'http://example.com/link_authors/1337/posts',
```

Comment:
if you wanna get wild, let's use a json api relationships link :)

- http://jsonapi.org/format/#fetching-relationships


```http
GET /link_authors/1/relationships/posts HTTP/1.1
Accept: application/vnd.api+json
```